bot/plotting/random_plot_generator.py

```diff
@@ -36,34 +41,65 @@ def random_plot_generator(
     while True:
 
         try:
-
-            models = [
-                pybamm.lithium_ion.DFN(),
-                pybamm.lithium_ion.SPM(),
-                pybamm.lithium_ion.SPMe(),
-            ]
-
-            model_num = random.randint(0, len(models) - 1)
-            model = models[model_num]
-
+            pybamm.set_logging_level("NOTICE")
             chemistries = [
-                # pybamm.parameter_sets.Ai2020,
+                pybamm.parameter_sets.Ai2020,
                 pybamm.parameter_sets.Chen2020,
                 pybamm.parameter_sets.Marquis2019,
+                pybamm.parameter_sets.Yang2017,
                 # pybamm.parameter_sets.Ecker2015,
                 # pybamm.parameter_sets.Ramadass2004,
             ]
 
             chem_num = random.randint(0, len(chemistries) - 1)
             chemistry = chemistries[chem_num]
 
+            particle_mechanics = ["swelling and cracking", "swelling only"]
+            sei = [
+                "ec reaction limited",
+                "reaction limited",
+                "solvent-diffusion limited",
+                "electron-migration limited",
+                "interstitial-diffusion limited",
+            ]
```

Comment:
"none" should be an option in these dicts too so that we have the case with only sei or only mechanics (but then check that they don't both choose "none" ...)